readthedocs/domains/templates/domains/notifications/pending_domain_configuration_site.html

```diff
@@ -0,0 +1,7 @@
+{% url "projects_domains_edit" domain.project.slug domain.pk as domain_url %}
+{% if domain.validation_process_expired %}
+  The validation period of your custom domain <a href="{{ domain_url }}">{{ domain.domain }}</a> has ended.
+  Save the domain to restart the process.
+{% else %}
+  The configuration of your custom domain <a href="{{ domain_url }}">{{ domain.domain }}</a> is pending.
```

Comment:
I think this line should communicate better than an action is required from the user.

Reply:
Something like `please complete the configuration` at the end?

Reply:
Yes. It could be something like

```suggestion
  The configuration of your custom domain <a href="{{ domain_url }}">{{ domain.domain }}</a> is pending. Go to <a>domain configuration</a> and follow the instructions to complete it.
```